### ISSUE TITLE
Increased saving throws item properties do not work in all cases.

### ISSUE THREAD

**Issue by YunimNWN** [Beamdog/nwn-issues#187]
**To Reproduce**

 - Load the attached module and type "/help" for chat commands.
 - [Test_Saves.zip](https://github.com/Beamdog/nwn-issues/files/5328948/Test_Saves.zip)
 - Type "/ring" to add rings with save bonus item properties and "/us *" to add save bonus effects.
 - Type "/fo", "/re", and "/wi" to test saves for Fortitude, Reflex, and Will respectively.

**Specifics**

 - OS: Windows
 - Game build: 8193.16
 - Distro platform: GOG
 - Content overrides? None
 - Regression? No, this occurred in 1.69 too.

**If needed, describe the bug**

This bug applies to both ItemPropertyBonusSavingThrow and ItemPropertyBonusSavingThrowVsX with nBonus > 12.
If the bonus is greater than 12 (ie. 13 to 20) then the item property doesn't work at all.
It doesn't increase saving throws, it doesn't apply to the +20 save bonus cap, and it doesn't apply to item property stacking.
The test module only has items with Fortitude and Universal save bonuses, but this bug applies to all save bonus item properties.

The module has a ring with +12 Fortitude saves and +11 Universal saves, which results in +12 Fortitude, +11 Reflex, +11 Will as expected.
There is also a ring with +13 Fortitude saves and +12 Universal saves, which results in +12 to all saves instead of +13 Fortitude, +12 Reflex, +12 Will.

There is no such issue with EffectSavingThrowIncrease, which works up to the +20 saving throw bonus cap.

**Comment by niv** [comment]
Apparently the `iprp_bonuscost.2da` is missing the required rows.

Try this one:

```
2DA V2.0

           Name   Label    Cost   Value
0          ****   Random   0      0
1          1045   1        1      1
2          1046   2        2      2
3          1047   3        3      3
4          1048   4        4      4
5          1049   5        5      5
6          1050   6        6      6
7          1051   7        7      7
8          1052   8        8      8
9          1053   9        9      9
10         1054   10       10     10
11         1954   11       11     11
12         1992   12       12     12
13         2113   13       13     13
14         2119   14       14     14
15         2139   15       15     15
16         2163   16       16     16
17         2164   17       17     17
18         2187   18       18     18
19         2191   19       19     19
20         2192   20       20     20

```

**Comment by Shad000w** [comment]
saving throws bonuses are limited to +12

there is no error in 2das as niv suggest, you can already put there higher values than +12 which you also demonstrated in the description, it just doesn't work

**Comment by YunimNWN** [comment]
Adding an edited iprp_bonuscost.2da file to my override folder fixed the issue, so it definitely fixes the issue in 8193.16.
